wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -716,3 +717,19 @@ def menu_props(context):
 @register.simple_tag
 def get_comments_enabled():
     return getattr(settings, 'WAGTAILADMIN_COMMENTS_ENABLED', True)
+
+
+@register.simple_tag
+def resolve_url_if_not_absolute(url):
```

Comment:
Django has a similar built-in function https://github.com/django/django/blob/4b6208ffdd93d5d379d247ec094d567ef208effd/django/shortcuts.py#L102

Have you considered using this?

Reply:
Ah, nice! Will update to use that...